tests/integration/cli/test_plugins.py

```diff
@@ -129,24 +129,28 @@ def test_list_does_not_repeat(ape_plugins_runner, installed_plugin):
     assert "ethereum" not in result.available_plugins
 
 
+@pytest.mark.skip(reason="Installation may replace local packages with unwanted versions")
 @github_xfail()
 def test_upgrade(ape_plugins_runner, installed_plugin):
     result = ape_plugins_runner.invoke(["install", TEST_PLUGIN_NAME, "--upgrade"])
     assert result.exit_code == 0
 
 
+@pytest.mark.skip(reason="Installation may replace local packages with unwanted versions")
```

Comment:
I think we should either

1. replace the `@github_xfail()` with this skip, could also store it as variable `github_skip = `
2. edit the `@github_xfail()` to include a manual marker (also registering it and stuff), this way allows us to run the tests manually if we want, which you may if you are editing the plugin commands a bunch

3. option 3 is delete the tests and only rely on the functional tests for plugin stufff.... but I kinda like having integration tests here, even if they don't run in CI

Reply:
What do you think about e7783af?

Reply:
It seems to skip them, but you want to add `@run_once` back in for those.
The integration tests are (oddly, imo) parametrized across projects.
To force an integ test to only run once (1 project) instead of n times you have to use `@run_once`.

Right now, when I run the module tests, it says this:

```
55 skipped, 3 xpassed, 1 warning
```

But if i import `run_once` and decorate them with that (note: github_xfail has this decor builtin to it), then it says:

```
 5 skipped, 3 xpassed, 1 warning
```

Reply:
Ah, didn't realize that.  A feature of the `@github_xfail` decorator lost.  Added.